accepted/2021/workloads/workload-installation.md

```diff
@@ -42,20 +42,20 @@ We have not yet settled on how workload installation will work when the .NET SDK
 
 The only workload that will be supported on Linux in .NET 6 will be Blazor AoT.  For .NET 6, we will recommend that developers using a .NET SDK installed via a package manager run `dotnet workload install` under `sudo`.  This will use the file based installation mechanism, ie it will download the workload packs and copy them to the .NET SDK folder.
 
-For source-built .NET, we should not be copying non source-built files into the source-built .NET SDK folder.  For that case, we will provide instructions for how to use workloads outside of the .NET SDK folder.  This will involve downloading and extracting various workload NuGet packages to a user folder, and setting some environment variables (`DOTNETSDK_WORKLOAD_MANIFEST_ROOTS` and `DOTNETSDK_WORKLOAD_PACK_ROOTS`) so that the .NET SDK knows where to find them.
+For source-built .NET, we should not be copying non source-built files into the source-built .NET SDK folder.  For that case, we will support [installing workloads to a user-local folder](https://github.com/dotnet/sdk/issues/18104) that does not require admin permission to write to.
 
 ## For .NET 7
 
-For .NET 7, we will need a better solution for source-built .NET, and may also want a better solution for package managers in general.  This will likely be one of the following options:
+For .NET 7, we will probably want a better option for package managers in general.  This will likely be one of the following options:
```

Comment:
I don't believe we have plans for workload pack in package managers for current workloads in the .NET 7 timeframe so the user-local is the only option at the moment.